README.md

```diff
@@ -65,3 +65,29 @@ You can set the file extension used by the UI to download a result from your fun
 * `environment`
 
 Set timeouts etc
+
+# OpenFaaS Template Store
+
+The Template Store are all the templates from OpenFaaS and OpenFaaS-Incubator organizations, along with user-defined templates which were approved by the community.
+
+## How to submit to the Template Store
+
+If you'd like to see your template in the official store follow these steps:
+
+1. Open issue with your template repository
```

Comment:
'Open an issue to propose the template and include a link to your template repository'
We should create an issue template to help extract the desired information.

Reply:
Done ✔️ 
> We should create an issue template to help extract the desired information.

I suppose we should open issue and separate PR for this?